sdk/identity/Azure.Identity/README.md

```diff
@@ -109,92 +86,137 @@ using Azure.Security.KeyVault.Secrets;
 ```c#
 using Azure.Identity;
 using Azure.Storage.Blobs;
 
-// authenticating a service principal with a client secret
-var credential = new ClientSecretCredential(tenantId, clientId, clientSecret);
+// when deployed to an azure host the default azure credential will authenticate the specified user assigned managed identity
+var credential = new DefaultAzureCredential(new DefaultAzureCredentialOptions { ManagedIdentityClientId = userAssignedClientId });
 
 var blobClient = new BlobClient(new Uri("https://myaccount.blob.core.windows.net/mycontainer/myblob"), credential);
 ```
 
-### Authenticating a service principal with a certificate
-This example demonstrates authenticating the `KeyClient` from the [Azure.Security.KeyVault.Keys][keys_client_library] client library using the `CertificateCredential`.
+### Define a custom authentication flow with the `ChainedTokenCredential`
+While the `DefaultAzureCredential` is generally the quickest way to get started developing applications for Azure, more advanced users may want to customize the credentials considered when authenticating. The `ChainedTokenCredential` enables users to combine multiple credential instances to define a customized chain of credentials. This example demonstrates creating a `ChainedTokenCredential` which will attempt to authenticate using managed identity, and fall back to authenticating via the Azure CLI if managed identity is unavailable in the current environment. The credential is then used to authenticate an `EventHubProducerClient` from the `Azure.Messaging.EventHubs` client library.
+
 ```c#
 using Azure.Identity;
-using Azure.Security.KeyVault.Keys;
-using System.Security.Cryptography.X509Certificates;
-
-// authenticating a service principal with a certificate
-var certificate = new X509Certificate2("./app/certs/certificate.pfx");
+using Azure.Messaging.EventHubs;
 
-var credential = new ClientCertificateCredential(tenantId, clientId, certificate);
+// authenticate using managed identity if it is available otherwise use the Azure CLI to auth
+var credential = new ChainedTokenCredential(new ManagedIdentityCredential(), new AzureCliredential());
 
-var keyClient = new KeyClient(new Uri("https://myvault.azure.vaults.net/"), credential);
+var eventHubProducerClient = new EventHubProducerClient("myeventhub.eventhubs.windows.net", "myhubpath", credential);
 ```
-### Authenticating a user with the default browser
-The `InteractiveBrowserCredential` allows an application to authenticate a user by launching the system's default browser. This example demonstrates authenticating the `BlobClient` from the [Azure.Storage.Blobs][blobs_client_library] client library using the `InteractiveBrowserCredential`.
 
-```c#
-using Azure.Identity;
-using Azure.Storage.Blobs;
+## Credential Classes
+### Authenticating Azure Hosted Applications
+|credential  | usage
```

Comment:
Use HTML table instead of markdown table for doc generation

Reply:
Discussed this offline and this is only needed for Java so it's rendered properly with javadoc.